es/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,342 @@
 
 <br>
 
 **43. Independent component analysis**
 
-&#10230;
+&#10230; Análisis de componentes independientes
 
 <br>
 
 **44. It is a technique meant to find the underlying generating sources.**
 
-&#10230;
+&#10230; Es una técnica destinada a encontrar las fuentes generadoras subyacentes.
 
 <br>
 
 **45. Assumptions ― We assume that our data x has been generated by the n-dimensional source vector s=(s1,...,sn), where si are independent random variables, via a mixing and non-singular matrix A as follows:**
 
-&#10230;
+&#10230; Suposiciones - Suponemos que nuestros datos x han sido generados por el vector fuente n-dimensional s=(s1,...,sn), donde si son variables aleatorias independientes; a través de una matriz A de mezcla y no singular, de la siguiente manera:
+
+&#10230; Suposiciones - Suponemos que nuestros datos x han sido generados por el vector fuente n-dimensional s=(s1,...,sn), donde si son variables aleatorias independientes; a través de una matriz A de mezcla y no singular, de la siguiente manera:
 
 <br>
 
 **46. The goal is to find the unmixing matrix W=A−1.**
 
-&#10230;
+&#10230; El objetivo es encontrar la matriz desmezcladora W=A−1.
 
 <br>
 
 **47. Bell and Sejnowski ICA algorithm ― This algorithm finds the unmixing matrix W by following the steps below:**
 
-&#10230;
+&#10230; Algoritmo ICA de Bell y Sejnowski - Este algoritmo encuentra la matriz desmezcladora W siguiente los siguientes pasos: 
 
 <br>
 
 **48. Write the probability of x=As=W−1s as:**
 
-&#10230;
+&#10230; Escribir la probabilidad de x=As=W−1s como:
 
 <br>
 
 **49. Write the log likelihood given our training data {x(i),i∈[[1,m]]} and by noting g the sigmoid function as:**
 
-&#10230;
+&#10230; Escriba la probabilidad dado nuestros datos de entrenamiento {x(i),i∈[[1,m]]} y denotando g, la función sigmoide, como:
 
 <br>
 
 **50. Therefore, the stochastic gradient ascent learning rule is such that for each training example x(i), we update W as follows:**
 
-&#10230;
+&#10230; Por lo tanto, la regla de aprendizaje de ascenso de gradiente estocástica es tal que para cada ejemplo de entrenamiento x(i), actualizamos W de la siguiente manera:
 
 <br>
 
 **51. The Machine Learning cheatsheets are now available in Spanish.**
 
-&#10230;
+&#10230; Aprendizaje no Supervisado ahora están disponibles en Español.
 
 <br>
 
 **52. Original authors**
 
-&#10230;
+&#10230; Autores Originales
 
 <br>
 
 **53. Translated by X, Y and Z**
 
-&#10230;
+&#10230; Traducido por X, Y y Z
 
 <br>
 
 **54. Reviewed by X, Y and Z**
 
-&#10230;
+&#10230; Revisado por X, Y y Z
 
 <br>
 
 **55. [Introduction, Motivation, Jensen's inequality]**
 
-&#10230;
+&#10230; [Introducción, Motivación, Desigualdad de Jensen]
 
 <br>
 
 **56. [Clustering, Expectation-Maximization, k-means, Hierarchical clustering, Metrics]**
 
-&#10230;
+&#10230; [Clusterización, Expectativa-Maximización, k-means, Agrupación jerárquica, Métricas]
```

Comment:
Use "_Agrupamiento (clustering en inglés)_". "Clusterización" is a very ugly anglicism. Furthermore, in "Hierarchical clustering" you translated it as "agrupamiento".